src/requests/request.ts

```diff
@@ -0,0 +1,12 @@
+import { PASSWORD, USERNAME } from "../constants";
```

Comment:
If you intend to put all the requests in one place I would recommend changing the file name and the function name appropriately, such that it represents the correct function there